src/Core/Services/CipherService.cs

```diff
@@ -1146,6 +1146,7 @@ private async Task EncryptCipherDataAsync(Cipher cipher, CipherView model, Symme
                     if (model.Login.Uris != null)
                     {
                         cipher.Login.Uris = new List<LoginUri>();
+                        model.Login.Uris.RemoveAll(u => u.Uri == null);
```

Comment:
@LRNcardozoWDF do we need to remove them? can't we just filter them in the foreach?
`model.Login.Uris.Where(u => u.Uri != null)`
Does the code work correctly if `u.Uri` is empty or whitespace?

Reply:
The result is the same either by removing or filtering, the item is saved without the empty or whitespaced URIs, do you think filter approach would be better?
Yes, it worked correctly for `u.Uri` empty or whitespace, for IVIE turned off and on.

Reply:
I think the code is simpler with the filtering and would have better performance as the list would have to be traversed once (with the remove, the list needs to be traversed twice, once for remove and the other one for the for loop). However, it would be really little gain performance wise unless the list is really long, which I don't think it'd be a case for login Uris. I'd go with filtering so we have less lines of code and less list mutations.
If you have any strong reason to go with `RemoveAll` over filtering please let me know, perhaps I'm missing something.

Reply:
Fixed in 06d1ad9587d10efe5ce3e7bcae9d1978f2e575f5